.appveyor.yml

```diff
@@ -0,0 +1,18 @@
+environment:
+  fast_finish: true
+  matrix:
+    - nodejs_version: "8"
+    - nodejs_version: "6"
+    - nodejs_version: "4"
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - npm install npm@latest -g
```

Comment:
Nitpick, but better to be verbose with flags in code so it's easier to understand:

```
- npm install --global npm@latest
```